webpack.config.js

```diff
@@ -174,7 +174,7 @@ if ( calypsoEnv === 'development' ) {
 	webpackConfig.entry.build = [
 		'webpack-dev-server/client?/',
 		'webpack/hot/only-dev-server',
-		path.join( __dirname, 'client', 'boot' )
+		path.join( __dirname, 'client', 'boot', 'app' )
```

Comment:
I renamed entry file to make diff look smaller, because GitHub doesn't handle well advanced rename chains. In my opinion we could even leave it as is now. If you disagree I would suggest perform rename just before we land this PR.